config/structs.go

```diff
@@ -1,14 +1,23 @@
 package config
 
+import "github.com/ossf/package-feeds/feeds"
+
+type PublisherConfig TypeConfigPair
+
 type ScheduledFeedConfig struct {
-	PubConfig    PublisherConfig `yaml:"publisher"`
-	EnabledFeeds []string        `yaml:"enabled_feeds"`
-	HTTPPort     int             `yaml:"http_port,omitempty"`
-	PollRate     string          `yaml:"poll_rate"`
-	Timer        bool            `yaml:"timer"`
+	PubConfig PublisherConfig `yaml:"publisher"`
+	Feeds     []FeedConfig    `yaml:"feeds"`
+	HTTPPort  int             `yaml:"http_port,omitempty"`
+	PollRate  string          `yaml:"poll_rate"`
+	Timer     bool            `yaml:"timer"`
 }
```

Comment:
The change here breaks configuration for anyone who has been using the `enabled_feeds` configuration option since added.

Keeping support for this is possible however it complicates the code unnecessarily as the migration to the new configuration is a minimal change.

```

enabled_feeds:
- pypi
```

Becomes 

```
feeds:
- type: pypi
```

Reply:
Speaking of breaking changes, there's an ongoing issue around if the value for the `Package{}` `type` field should be changed, or even the keyword itself https://github.com/ossf/package-feeds/issues/93

If the latter is the case then this may inadvertently require another breaking change to this option.